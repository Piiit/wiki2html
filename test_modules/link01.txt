Simple link with protocol: [[http://www.google.de]]
Simple link without protocol: [[www.unibz.it]]
Link with a name (piped): [[www.unibz.it|Uni BZ]]
Link with a name (space): [[www.unibz.it Uni BZ]]

* Link as a list item: [[Testing]]
* Next item
