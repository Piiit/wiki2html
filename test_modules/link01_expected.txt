Simple link with protocol: <a href='http://www.google.de'>http://www.google.de</a>
Simple link without protocol: <a href='www.unibz.it'>www.unibz.it</a>
Link with a name (piped): <a href='www.unibz.it'>Uni BZ</a>
Link with a name (space): <a href='www.unibz.it'>Uni BZ</a>

<ul>
<li>Link as a list item: <a href='Testing'>Testing</a></li>
<li>Next item</li>
</ul>
